packages/bruno-app/src/components/Sidebar/NewRequest/index.js

```diff
@@ -58,6 +60,12 @@ const NewRequest = ({ collection, item, isEphemeral, onClose }) => {
           name: 'requestName',
           message: `The request names - collection and folder is reserved in bruno`,
           test: (value) => {
+            if (isWindowsOS() && variableNameRegex.test(value) === false) {
+              toast.error(
+                `Variable contains invalid characters! Variables must only contain alpha-numeric characters, "-", "_", "."`
```

Comment:
This is the message from Bru.js for variables I think. Please make this more unique and use request name instead of "variable name"

Reply:
Hey, thanks for reviewing, I just updated the message :)